src/utils/ConfigHelper.ts

```diff
@@ -141,6 +141,57 @@ const configs: ConfigHelperConfig[] = [
     dispenserAddress: null,
     metadataContractAddress: null,
     startBlock: 90707
+  },
+  {
+    networkId: 2021000,
+    network: 'gaiaxtestnet',
+    nodeUri: 'https://gaia-x.rpc',
+    metadataCacheUri: 'https://aquarius.gaiaxtestnet.oceanprotocol.com',
+    providerUri: 'https://provider.gaiaxtestnet.oceanprotocol.com',
+    subgraphUri: 'https://subgraph.gaiaxtestnet.oceanprotocol.com',
+    explorerUri: 'https://gaiaxtestnet.explorer',
+    oceanTokenAddress: null,
+    oceanTokenSymbol: 'OCEAN',
+    factoryAddress: null,
+    poolFactoryAddress: null,
+    fixedRateExchangeAddress: null,
+    dispenserAddress: null,
+    metadataContractAddress: null,
+    startBlock: 0
+  },
+  {
+    networkId: 80001,
+    network: 'mumbai',
+    nodeUri: 'https://polygon-mumbai.infura.io/v3/',
+    metadataCacheUri: 'https://aquarius.mumbai.oceanprotocol.com',
+    providerUri: 'https://provider.mumbai.oceanprotocol.com',
+    subgraphUri: 'https://subgraph.mumbai.oceanprotocol.com',
+    explorerUri: 'https://explorer-mumbai.maticvigil.com/',
```

Comment:
It says right on top:

> This explorer will be deprecated soon, please use PolygonScan.com

So we should also switch main polygon explorer to https://polygonscan.com. Could not figure out yet how to switch to Mumbai on there though.